src/main/java/com/myaudiolibrary/web/model/Artist.java

```diff
@@ -0,0 +1,89 @@
+package com.myaudiolibrary.web.model;
+
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+
+@Entity
+@Table(name = "artist")
+public class Artist
+{
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "ArtistId")
+    private long id;
+
+    @Column(name = "Name",nullable = false)
+//    @NotNull
+    private String name;
+
+    @OneToMany(mappedBy = "artist",fetch = FetchType.EAGER,cascade = CascadeType.REMOVE)//orphanRemoval = true
```

Comment:
Le `CascadeType.ALL` est utilisable mais il faut bien ajouter le `orphanRemoval` afin que la suppression de l'album soit bien persistée. Le `CascadeType.REMOVE` marche bien aussi dans ce cas

Plus d'infos [ici](https://openjpa.apache.org/builds/2.4.0/apache-openjpa/docs/jpa_overview_meta_field.html#jpa_overview_meta_cascade) et [là](https://stackoverflow.com/questions/306144/jpa-cascadetype-all-does-not-delete-orphans)